gameStateFile.json

```diff
@@ -0,0 +1,35 @@
+[
+  [
+    {
+      "value": 1

```

Comment:
I don't usually discuss formatting, but the limit to 10 columns makes me wonder why you chose everyone to scroll a lot while we have large screens ? ;)

Reply:
Seems like it's the default behaviour with prettier ! Do we need to change it ?

Reply:
no, if prettier does it, let's keep it.